File: myItem/qqrobot3/src/main/java/pers/fj/robot/core/listener/MyListener.java
package pers.fj.robot.core.listener;

import love.forte.simbot.annotation.OnPrivate;
import love.forte.simbot.api.message.events.PrivateMsg;
import love.forte.simbot.api.sender.MsgSender;
import org.springframework.stereotype.Component;

@Component
public class MyListener {

    /**
     * 监听并且复读
     * @param privateMsg
     * @param msgSender
     */
    @OnPrivate
    public void onEvent(PrivateMsg privateMsg, MsgSender msgSender) {
        // do...
        msgSender.SENDER.sendPrivateMsg(privateMsg,privateMsg.getMsgContent());
    }

}

File: myItem/qqrobot2/src/main/kotlin/feijiang/robot/core/listener/MyListener.kt
package feijiang.robot.core.listener


import love.forte.simboot.annotation.Listener
import love.forte.simbot.event.FriendMessageEvent
import org.springframework.stereotype.Component


@Component
class MyListener {

    @Listener
    suspend fun FriendMessageEvent.onEvent() {
        // do..
        println("xxx")
    }
}